studio/src/pages/[organizationSlug]/webhooks.tsx

```diff
@@ -0,0 +1,214 @@
+import { EmptyState } from "@/components/empty-state";
+import { getDashboardLayout } from "@/components/layout/dashboard-layout";
+import { Button } from "@/components/ui/button";
+import { Input } from "@/components/ui/input";
+import { Checkbox } from "@/components/ui/checkbox";
+import {
+  Form,
+  FormControl,
+  FormDescription,
+  FormField,
+  FormItem,
+  FormLabel,
+  FormMessage,
+} from "@/components/ui/form";
+import { Loader } from "@/components/ui/loader";
+import { useToast } from "@/components/ui/use-toast";
+import { SubmitHandler, useZodForm } from "@/hooks/use-form";
+import { NextPageWithLayout } from "@/lib/page";
+import { ExclamationTriangleIcon } from "@heroicons/react/24/outline";
+import { useMutation, useQuery } from "@tanstack/react-query";
+import { EnumStatusCode } from "@wundergraph/cosmo-connect/dist/common_pb";
+import {
+  getOrganizationWebhookConfig,
+  saveOrganizationWebhookConfig,
+} from "@wundergraph/cosmo-connect/dist/platform/v1/platform-PlatformService_connectquery";
+import { z } from "zod";
+
+const FormSchema = z.object({
+  endpoint: z.string().url(),
+  key: z.string().optional(),
+  events: z.array(z.string()).optional(),
+});
+
+type Input = z.infer<typeof FormSchema>;
+
+const events = [
+  { id: "graph.schema.updated", label: "Graph Schema Update" },
```

Comment:
Can we sync this with our proto definition?

Reply:
Yep. Done